docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -78,6 +78,8 @@ If JBang has already been installed, you can directly use it:
 ----
 # This can also be used to update to the latest version
 jbang app install --fresh --force quarkus@quarkusio
+# Or create an alias in order to use a specific version
+jbang app install --name qs2.2.5 io.quarkus:quarkus-cli:2.2.5.Final:runner
```

Comment:
Please make it a separate block. It's confusing otherwise if people don't read and just copy paste.